manifests/deploy-both-cluster-and-namespaced-scoped.yaml

```diff
@@ -0,0 +1,60 @@
+# This is the cluster scoped example - in k3s/RKE2 this is unnecessary as it will be deployed out of the box.
+# To use this example on k3s/RKE2 you should exclude this part, or disable the embedded controller.
```

Comment:
I'm confused what's going on with this example - we deploy one controller to the default namespace that watches all namespaces, and another to the helm-controller namespace that watches only its own namespace?

Unless I'm missing something this doesn't seem like a valid configuration. You could run one that watches all namespaces, or multiple that watch specific namespaces, but the two should not be mixed.

Reply:
>You could run one that watches all namespaces, or multiple that watch specific namespaces, but the two should not be mixed otherwise the global one will still step on the namespaced ones.

That is essentially the behavior this is seeking to modify - expanding on the prior works around the ManagedBy field to allow co-existence of multiple controllers. Either of `helm-controller` type deployment - or when the controller is re-used as a library as with PrometheusFederator.